package.json

```diff
@@ -84,7 +84,7 @@
     "start-joplin-prod": "NODE_PATH=./src CMS_API='http://joplin.herokuapp.com/api/graphql' CMS_MEDIA='https://joplin-austin-gov-static.s3.amazonaws.com/staging/media' CMS_DOCS='https://joplin-austin-gov-static.s3.amazonaws.com/production/media/documents' npm-run-all -p watch-css start-js",
     "start-joplin-staging": "NODE_PATH=./src CMS_API='http://joplin-staging.herokuapp.com/api/graphql' CMS_MEDIA='https://joplin-austin-gov-static.s3.amazonaws.com/staging/media' CMS_DOCS='multiple' npm-run-all -p watch-css start-js",
     "start-joplin-pr": "NODE_PATH=./src CMS_API='https://joplin-pr-import-everything.herokuapp.com/api/graphql' CMS_MEDIA='https://joplin-austin-gov-static.s3.amazonaws.com/staging/media' CMS_DOCS='multiple' npm-run-all -p watch-css start-js",
-    "start-joplin-local": "NODE_PATH=./src CMS_API='http://localhost:8000/api/graphql' CMS_MEDIA='http://localhost:8000/media' CMS_DOCS='multiple' npm-run-all -p watch-css start-js",
```

Comment:
i have been able to do start-joplin-local this whole time

Reply:
Really! Hmm. Maybe it was just breaking on full site builds.